charts/hono/values.yaml

```diff
@@ -328,7 +334,10 @@ adapters:
     enabled: true
     # imageName contains the name (including registry and tag)
     # of the container image to use for the AMQP adapter
-    imageName: index.docker.io/eclipse/hono-adapter-amqp-vertx:1.2.3
+    imageName: index.docker.io/eclipse/hono-adapter-amqp-vertx
+    # imageTag contains the tag of the container image to deploy.
+    # If not specified, the value of the honoImagesTag property is used.
+    # imageTag:
```

Comment:
According to what is used in the individual component yaml files and the README, this has to be named `tagName` instead of `imageTag`.

To verify:
````
helm install --dry-run --debug --dependency-update -n hono --set honoImagesTag=1.3.0-M3 --set adapters.amqp.tagName=1.2.3 eclipse-hono hono/ | grep "\- image:"
````
gives the expected results.

But
````
helm install --dry-run --debug --dependency-update -n hono --set honoImagesTag=1.3.0-M3 --set adapters.amqp.imageTag=1.2.3 eclipse-hono hono/ | grep "\- image:"
````
doesn't.


I would prefer the term `imageTag` here actually, which would mean changing individual component yaml files and the README accordingly.

Reply:
thanks for spotting. I have no clue why it ended up this way in the deployment descriptors. It should definitely be *imageTag* (consistent with *imageName*). I'll fix that accordingly ...